cli/README.adoc

```diff
@@ -10,6 +10,60 @@ Download the archive file, unpack and run:
     cd garage-sign/
     bin/garage-sign --help
 
+== Delegations
+
+There are two actors in a delegations use case. The OEM, which owns the remote repository managed by ota-tuf, and the Supplier, which will sign metadata about a component they build.
+
+The OEM runs garage-sign against a local repository initialized using `garage-sign init`. The changes can then be pushed as normal to a remote `ota-tuf` repository.
+
+The Supplier, runs garage-sign but it does not have local repository, instead the supplier edits metadata files using garage-sign. The resulting files can then be sent to the OEM using a secure channel (e.g. encrypted email), which adds that data to their local and remote repositories.
+
+The basic use case for delegations works as follows.
+
+1. The OEM manages a local repository as normal:
+
+   garage-sign init --repo paris -c credentials-paris.zip
```

Comment:
Is this documentation for "us" or for a wider audience? Because in the second case I could imagine someone not aware of our hats wondering "what is this paris naming about".

Reply:
This is for us for the moment.